src/WebAuthn.sol

```diff
@@ -122,47 +109,41 @@ library WebAuthn {
             return false;
         }
 
-        // 11. and 12. will be verified by the signature check
-        // 11. Verify that the value of C.type is the string webauthn.get.
+        string memory _type = webAuthnAuth.clientDataJSON.slice(webAuthnAuth.typeIndex, webAuthnAuth.typeIndex + 21);
```

Comment:
NIT: magic number `21` here should be described in a comment. I.E. 

// bytes of string `"type":"webauthn.get"`

Reply:
thanks, updated!